src/filetest/regalloc.rs

```diff
@@ -44,11 +44,12 @@ impl SubTest for TestRegalloc {
         let mut comp_ctx = cretonne::Context::new();
         comp_ctx.func = func.into_owned();
 
-        comp_ctx.flowgraph();
+        comp_ctx.cfg.compute(&comp_ctx.func);
         // TODO: Should we have an option to skip legalization?
         comp_ctx.legalize(isa).map_err(|e| {
             pretty_error(&comp_ctx.func, context.isa, e)
         })?;
+        comp_ctx.domtree.compute(&comp_ctx.func, &comp_ctx.cfg);
```

Comment:
It seems unfortunate to have these direct calls to the passes spread all over the testing code. How about adding argument-less methods like `comp_ctx.ensure_cfg()`, and `comp_ctx.ensure_domtree()`?